pkg/querier/worker/frontend_processor_test.go

```diff
@@ -39,7 +39,7 @@ func TestRecvFailDoesntCancelProcess(t *testing.T) {
 		running.Store(true)
 		defer running.Store(false)
 
-		mgr.processQueriesOnSingleStream(ctx, cc, "test:12345")
+		mgr.processQueriesOnSingleStream(ctx, cc, "test:12345", "1")
```

Comment:
is `"1"` necessary?

Reply:
You are right. Since it's ignored we don't need it here.